### PR TITLE
V030

### PR DESCRIPTION
**Pull request by atayarani**
@marcuswalser @Tapjoy/eng-group-ops 

Notes:
* This code is in transition. I had to leave some extra cruft around, so other parts didn't break.
* Tested by creating and deleting a user.

### CODE REVIEW

File: lib/tapjoy/ldap/user/create.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          case opts[:type]
+          when 'user'
+            ou = 'People'
+          when 'service'
+            ou = Tapjoy::LDAP::client.service_ou
+          else
+            puts 'Unknown type'
+          end
+
+          dn = "uid=#{ username },ou=People,#{ Tapjoy::LDAP::client.basedn }"
```
Comment (ehealy) [inline lib/tapjoy/ldap/user/create.rb]: based on the case statement above it looks like `ou=People` should be `ou=#{ou}`

**Comment by ehealy** [comment]
One potential minor issue, otherwise this looks sane to me.

**Comment by atayarani** [comment]
wow, that's been a bug in the released code for quite some time.

On Fri, Dec 4, 2015 at 9:03 PM, ehealy <notifications@github.com> wrote:

> One potential minor issue, otherwise this looks sane to me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Tapjoy/ldap_tools/pull/3#issuecomment-162127934>.
>



-- 
Ali Tayarani || Senior Engineer || *Tapjoy* <http://tapjoy.com/> ||
@sysadmali <https://twitter.com/sysadmali>

**Comment by atayarani** [comment]
@ehealy fixed

**Comment by ehealy** [comment]
LGTM

**Pull request merged**